ci-scripts/package.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
```

Comment:
What are plans for this file. Do we want to have that in all repos? Do we want to have that built into architect? Or maybe we want to make it a cricleci orb.

Reply:
yes, we need to sync with VOO and sig releng on this matter in general.

+1 on having it somewhere central.

We also talked about checking out GH actions, and see if we really need the external dependency on circle CI (even as orb) for this simple functionality or if it living within GH would make more sense.

Reply:
Investigating GH actions and Orbs makes sense to me. As then maybe we don't need the file. I agree we need to sync with VOO and sig-releng here.

Reply:
and it is ok to have it first in a script/file and then move that around to a better implementation (also with more experience on where the process does not work for us in real life then)